timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/server/databasemigration/WwPersonSortIndexesDatabaseMigration.java

```diff
@@ -56,7 +55,7 @@ public void applyToVertex(Vertex vertex) throws IOException {
             .orElseGet(() -> Try.success(new String[0]))
             .getOrElse(() -> new String[0]));
 
-    if (types.contains("person")) {
+    if (types.contains("wwperson")) {
```

Comment:
Als we de code van de indexer aanpassen dan verandert de werking van deze migratie. Dit is daarmee dus niet echt een migratie die de database aanpast van staat A naar staat B, maar meer een die als een idempotent stapje op de database geapplied wordt. Zie ik dat goed?